demo/test/unit/test_desc_event.gd

```diff
@@ -73,3 +73,18 @@ class TestEventDescription:
 	func test_assert_sound_size():
 		var desiredValue: float = 2.0
 		assert_eq(Fmod.desc_get_sound_size("event:/Vehicles/Car Engine"), desiredValue, "Event description sound size should be " + str(desiredValue))
+		
+	func test_assert_should_retrieve_user_property_by_name():
+		var desiredSize: int = 0
```

Comment:
I am not sure if these events contain any properties. I have not found an actual FMOD project in this repo to check on it. So I am just making sure that calling these methods themselves parses correctly and does not cause any errors.

Reply:
We kinda just took the bank samples provided by FMOD for our demo so I am not sure if there is any user property to retrieve in those. But just checking if the methods doesn't return any error is already enough imo.